hircluster.c

```diff
@@ -63,8 +63,6 @@
 #define REDIS_COMMAND_ASKING "ASKING"
 #define REDIS_COMMAND_PING "PING"
 
-#define REDIS_PROTOCOL_ASKING "*1\r\n$6\r\nASKING\r\n"
```

Comment:
I understand this one is unused too. But wouldn't it be more efficient to use this pre-encoded ASKING rathar than `redisCommand(c, REDIS_COMMAND_ASKING);` as we do currently?

Reply:
I should be better, there are 4 commands used a couple of lines above that can be changed.
It would require some different usage of the hiredis api and for async there is a `redisAsyncFormattedCommand()`, but its not as straight forward for blocking what I know. It should be looked into.